.psh.yml.dist

```diff
@@ -1,3 +1,29 @@
+header: |
```

Comment:
please rename to .psh.yml
it prevents errors like this one
because psh only reads the psh.yml.dist if theres no psh.yml